models/disk.go

```diff
@@ -56,6 +61,56 @@ func CreateDisk(cdm CreateDiskMetadata) Disk {
 	return disk
 }
 
+func MeasureDiskThroughput(d Disk) int {
```

Comment:
moglibyśmy to wrzucić jako metoda dysku? Niech to będzie zaimplementowane w AbstractDisk :D

Reply:
Moim zdaniem nie da się tak zrobić. Początkowo implementowałem to w AbstractDisk, ale o ile mi wiadomo, nie można w Go potem z poziomu AbstractDisk wywołać metody Upload do rzeczywistego dysku dziedziczącego po AbstractDisk - stąd to rozwiązanie z Interface Disk.

Reply:
jasne